webapp/channels/src/components/user_settings/modal/user_settings_modal.tsx

```diff
@@ -275,17 +309,25 @@ class UserSettingsModal extends React.PureComponent<Props, State> {
 
     render() {
         const {formatMessage} = this.props.intl;
-        if (this.props.currentUser == null) {
-            return (<div/>);
-        }
 
-        const modalTitle = this.props.isContentProductSettings ? formatMessage({
-            id: 'global_header.productSettings',
-            defaultMessage: 'Settings',
-        }) : formatMessage({
-            id: 'user.settings.modal.title',
-            defaultMessage: 'Profile',
-        });
+        let modalTitle: string;
+
+        if (this.props.adminMode && this.props.currentUser) {
+            modalTitle = formatMessage({
+                id: 'userSettings.adminMode.modal_header',
+                defaultMessage: "{userDisplayName}'s Settings",
+            }, {
+                userDisplayName: getDisplayName(this.props.currentUser),
+            });
+        } else {
+            modalTitle = this.props.isContentProductSettings ? formatMessage({
```

Comment:
Nit, these should be turned into an else if instead of having a ternary inside the else case

Reply:
Done.